_docs/developer/troubleshooting/installation_troubleshooting.md

```diff
@@ -155,6 +155,13 @@ with no explanation, then there are a couple of things that may be going wrong:
   ```
   VM_PORT_SITE=1512 vagrant up
   ```
+  *  Steps to edit the env file
+  
+  1. Go to Submitty install directory and locate the file named "Vagrantfile."
+  2. Open the "Vagrantfile" in a text editor.
```

Comment:
Developers should not manually edit the Vagrantfile. Doing so will make it difficult for git commit tracking.

Reply:
This is indeed true, unless dev puts it in git ignore, it can mess with tracking. On my second thought, I think we should revert this change entirely.

Reply:
Yeah, it's true that it will make tracking a little bit difficult, but they should be aware of how they can create the env file.


We can do one thing: add one more line about that to put it in gitignore or not to commit it as a note.

Reply:
Gitignore is also a tracked file, so you will run into even more issues by editing it.
We already have instructions on creating the env file, see the last bullet under https://submitty.org/developer/troubleshooting/installation_troubleshooting#forwarded-port-already-in-use
It used to be the first bullet under the heading, but it looks like it got pushed down by later additions.

Reply:
Hmm, let me remove it.

Reply:
Yeah, I get it.